src/chrome/content/rules/MERY.jp.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+
+	Problematic subdomains:
+
+		- assets	(akamai)
+		- bravo	(akamai)
+		- cashmere	(akamai)
+		- content	(akamai)
+		- galena	(akamai)
```

Comment:
Don't we need to remove traffics through Akamai? https://github.com/EFForg/https-everywhere/pull/6100#issuecomment-238074839
I'm confused here :-/